src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataServiceSpec.scala

```diff
@@ -412,7 +351,7 @@ class FileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Matcher
     val addFileMetadata = addFileMetadataCaptor.getValue
     val fileMetadataDelete = fileMetadataDeleteCaptor.getValue
 
-    addFileMetadata.size should equal(4)
+    addFileMetadata.size should equal(2)
```

Comment:
Why has the expected size changed here?

Reply:
The expected size changed because I removed the duplicate `closureTypeField` from the `mockFields` variable in the `FileMetadataServiceSpec`